frontend/src/components/profile/ProfileForm.jsx

```diff
@@ -1,32 +1,91 @@
 // NPM Packages
 import React, { useState, useEffect } from "react";
 
-export default function ProfileForm({ desc, onSubmit }) {
+// Project Files
+import ResponsiveTextArea from "../ResponsiveTextArea";
+
+export default function ProfileForm({
+  userDescription,
+  userName,
+  userAccountType,
+  onSubmit,
+}) {
   // Local state
   const [description, setDescription] = useState("");
+  const [name, setName] = useState("");
+  const [accountType, setAccountType] = useState("");
```

Comment:
Can you just set the "pet" as initial state, because "pet" is first option and to setAccountType you need to click it, but nobody will click it, because it's already there and then it'll not update to pet...

Reply:
Cheers, done